modules/oc-mirror-command-reference.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected.adoc
+
+:_content-type: REFERENCE
+[id="oc-mirror-command-reference_{context}"]
+= Command reference for oc-mirror
+
+The following tables describe the `oc mirror` subcommands and flags:
+
+.oc mirror subcommands
+[cols="1,2",options="header"]
+|===
+|Subcommand
+|Description
+
+|`completion`
+|Generate the autocompletion script for the specified shell.
+
+|`describe`
+|Output the contents of an image set.
+
+|`help`
+|Show help about any subcommand.
+
```

Comment:
The `init` parameter and description is missing when compared to the same table as 4.11 docs. Maybe this is intentional?



init | Output an initial image set configuration template.
-- | --





init | Output an initial image set configuration template.
-- | --

Reply:
Yes that's correct. --init was introduced in 4.11.